Makefile

```diff
@@ -48,3 +49,9 @@ setup:
 .PHONY: test
 test: dist/ramda.js
 	npm test
+	find src -name '*.js' -not -path 'src/internal/*' \
+	| sed 's:src/\(.*\)[.]js:exports.\1 = require("./src/\1");:' >index.js
+	sed -i '' '/"main":/d' package.json
+	$(MOCHA) -- $(shell find test -name '*.js' -not -name 'installTo.js' -not -name 'test.examplesRunner.js')
```

Comment:
Run the test suite (a second time), skipping tests for `R.installTo` since this function requires access to `R` which is not available.